src/Console/Command/SlackLoggerAwareTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace duckpony\Console\Command;
+
+use Monolog\Handler\SlackHandler;
+use Monolog\Logger;
+
+trait SlackLoggerAwareTrait
+{
+    public function initSlackLogger(Logger $logger, array $config): void
+    {
+        $slackToken   = $config['Logger']['Slack']['Token'];
```

Comment:
This would result in a notice about undefined index if the config wasn't properly updated.
Either raise a proper exception to notify the user that the config is not correct or simply use null coalesce to avoid notice.

Reply:
Very true

Reply:
Usinf null coalesce because the parameters are optional.